easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
log.error does something different then log.debug, log.warning, log.exception or log.info, it's an entire different kind of function, this is where you run into unformatted log messages, because it does things differently. This is the correct and recommended way of logging things, it will improve performance on non debug runs, because the string interpolation never has to happen.